src/components/CaptureMatching/CaptureMatchingFilter.js

```diff
@@ -0,0 +1,236 @@
+import React, { useContext } from 'react';
```

Comment:
This is fine to get us started, but I think we need to do some refinement of the UI.
I would expect the filter to appear as a drawer on top of everything else, similar to Capture Details or Grower Details, and the styling should be similar to the filter for Earnings and Payments.

This toggled style could work, but would need to be closer on the screen to the thing it's filtering.